test/admin/dev.test.js

```diff
@@ -97,3 +97,55 @@ describe('Dev V2', () => {
     ]);
   });
 });
+
+const FUNC_NAME = 'myFunction';
+const FUNC_SOURCE = 'exports = function(arg1, arg2){ return {sum: arg1 + arg2, userId: context.user.id}}';
+const myFunction = () => ({ name: FUNC_NAME, source: FUNC_SOURCE });
+
+async function createTestFunction(functions) {
+  let func = await functions.create(myFunction());
+  expect(func.name).toEqual(FUNC_NAME);
+
+  let funcs = await functions.list();
+  expect(funcs).toHaveLength(1);
+  expect(funcs[0].name).toEqual(FUNC_NAME);
+}
+
+describe('Dev Function', () => {
+  let test = new StitchMongoFixture();
+  let apps;
+  let app;
+  let user;
+  let dev;
+
+  beforeAll(() => test.setup());
+  afterAll(() => test.teardown());
+
+  beforeEach(async () => {
+    let adminClient = await getAuthenticatedClient(test.userData.apiKey.key);
+    test.groupId = test.userData.group.groupId;
+    apps = await adminClient.v2().apps(test.groupId);
+    app = await apps.create({ name: 'testname' });
+    dev = adminClient.v2().apps(test.groupId).app(app._id).dev();
+
+    await createTestFunction(adminClient.v2().apps(test.groupId).app(app._id).functions());
+    await createUserPassProvider(adminClient.v2().apps(test.groupId).app(app._id).authProviders());
+    user = await createAppUser(adminClient.v2().apps(test.groupId).app(app._id).users());
+  });
+
+  afterEach(async () => {
+    await apps.app(app._id).remove();
+  });
+
+  it('Supports executing the function', async () => {
+    const result = await dev.executeFunction(
+      user._id,
+      FUNC_NAME,
+      777,
+      23,
+    );
+
+    expect(result.sum).toEqual({'$numberInt': '800'});
```

Comment:
nit: you could just combine these into a single `expect` on the entire object, for extra strictness.